Lib/typing.py

```diff
@@ -671,19 +678,19 @@ def Union(self, parameters):
       type(None).
     - Unions of unions are flattened, e.g.::
 
-        assert Union[Union[int, str], float] == Union[int, str, float]
+        >>> assert Union[Union[int, str], float] == Union[int, str, float]
 
     - Unions of a single argument vanish, e.g.::
 
-        assert Union[int] == int  # The constructor actually returns int
+        >>> assert Union[int] == int  # The constructor actually returns int
 
     - Redundant arguments are skipped, e.g.::
 
-        assert Union[int, str, int] == Union[int, str]
+        >>> assert Union[int, str, int] == Union[int, str]
 
     - When comparing unions, the argument order is ignored, e.g.::
 
-        assert Union[int, str] == Union[str, int]
+        >>> assert Union[int, str] == Union[str, int]
```

Comment:
Could we leave these ones the way they are? I feel like the repeated `>>>` prompts add a fair amount of visual clutter here, because of the fact that it's only ever one line at a time in each session:

https://github.com/python/cpython/blob/7b835b754e8a892a0bbb387cd71462574f6d3420/Lib/typing.py#L675-L694

The most important thing here is to keep the docstrings readable for people browsing the source code or calling `help()` in the REPL :-)